docs/api/session.md

```diff
@@ -311,6 +311,32 @@ session.fromPartition('some-partition').setPermissionRequestHandler((webContents
 })
 ```
 
+#### `ses.setPermissionCheckHandler(handler)`
+
+* `handler` Function<Boolean> | null
+  * `webContents` [WebContents](web-contents.md) - WebContents checking the permission.
+  * `permission` String - Enum of 'media'.
+  * `requestingOrigin` String - The origin URL of the permission check
+  * `details` Object - Some properties are only available on certain permission types.
```

Comment:
the code seems to unconditionally set both properties. Which ones aren't available on certain permission types?

Reply:
@ckerr This is future documentation so that when we add new permission types (currently this method only has one) it doesn't catch people by surprise 👍